modules/custom/openy_programs_search/src/Plugin/Block/ProgramsSearchBlock.php

```diff
@@ -93,15 +128,26 @@ public function blockForm($form, FormStateInterface $form_state) {
 
     $form = parent::blockForm($form, $form_state);
     $conf = $this->getConfiguration();
+    $request = $this->requestStack->getSchemeAndHttpHost();
+    $url = "{$request}/admin/openy/integrations/daxko/programs-search";
 
     $form['locations_config'] = [
       '#type' => 'details',
       '#title' => $this->t('Locations Config'),
     ];
 
+    try {
+      $locations = $this->storage->getLocations();
+    }
+    catch (\Exception $e) {
+      $locations = [];
+      $this->logger->error($e->getMessage());
+      $this->messenger->addError($this->t("Сan't fetch Daxco data, please re-check your settings {$url}"));
```

Comment:
Please use t() and placeholders to insert the URL

Reply:
not fixed

Reply:
fixed